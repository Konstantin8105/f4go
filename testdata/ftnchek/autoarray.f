      program main
      call test(100)
      end
      subroutine test(n)
      integer n
C the following local array cannot have variable bounds.
      integer auto(n)
      auto(1) = 5
      auto(2) = auto(1)*2
      end

