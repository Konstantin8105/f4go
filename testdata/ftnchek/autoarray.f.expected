token(97)           	program
IDENT               	main
token(105)          	call
IDENT               	test
(                   	(
INT                 	100
)                   	)
token(102)          	end
token(96)           	subroutine
IDENT               	test
(                   	(
IDENT               	n
)                   	)
token(98)           	integer
IDENT               	n
COMMENT             	C the following local array cannot have variable bounds.
token(98)           	integer
IDENT               	auto
(                   	(
IDENT               	n
)                   	)
IDENT               	auto
(                   	(
INT                 	1
)                   	)
=                   	=
INT                 	5
IDENT               	auto
(                   	(
INT                 	2
)                   	)
=                   	=
IDENT               	auto
(                   	(
INT                 	1
)                   	)
*                   	*
INT                 	2
token(102)          	end
